components/Footer.js

```diff
@@ -1,14 +1,46 @@
 import React from 'react';
+import FooterLink from './FooterLink';
 
-const Footer = () => {
+const FooterWrapper = (props) => {
+  const style = {
+    backgroundColor: '#D2D3DC',
+    border: 'solid 1px #D2D3DC',
+    boxShadow: '0 0 5px #777',
+    padding: 15,
```

Comment:
It's all good.  It's a convention that gets used a lot, but it's almost never commented on.